### PR TITLE
Switched to sass

### PR DESCRIPTION
**Pull request by tonywu315**

### CODE REVIEW

File: constellations/gemini-admin/src/app.sass [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import "_.constants.sass"
+
+body
+    margin: 0
+    font-family: $fontFamilyRoboto
+    color: $colorBlackDark
```
Comment (mythicat554) [inline constellations/gemini-admin/src/app.sass:6]: I think color should come before font-family?

File: constellations/gemini-admin/src/header/header.sass [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        list-style: none
+        overflow: hidden
+        transition: 0.1s linear height
+
+        & li
+            height: $listHeight
+            line-height: $listHeight
+            padding: 0 $spacing2x
+            text-align: center
+            background-color: $colorGrayLight
```
Comment (mythicat554) [inline constellations/gemini-admin/src/header/header.sass:71]: similar thing here: background-color before text-align

File: constellations/gemini-admin/src/modals/modal.js [hunk 3]
```diff
@@ -1,22 +1,16 @@
...
 const classnames = require("classnames");
 
 export class Modal extends React.Component {
-    constructor(props) {
-        super(props);
-        this.handleDismiss = this.handleDismiss.bind(this);
-    }
-
-    handleDismiss() {
+    handleDismiss = () => {
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/modals/modal.js:7]: oh nice! thanks

File: constellations/gemini-admin/src/modals/modal.sass [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    &.show
+        opacity: 1.0
+        visibility: visible
+
+.modal-overlay
+    position: fixed
+    top: 0
+    left: 0
+    z-index: zIndex
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/modals/modal.sass:17]: `$zIndex` ?

File: constellations/gemini-admin/src/modals/modal.sass [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    &.show
+        opacity: 0.5
+        visibility: visible
+
+.modal
+    position: fixed
+    top: 50%
+    left: 50%
+    transform: translate(-50%, -50%)
+    z-index: zIndex
```
Comment (ahsu1230) [inline constellations/gemini-admin/src/modals/modal.sass:33]: $zIndex?

**Pull request merged**